frontend/src/queries/nodes/DataTable/ColumnConfigurator/ColumnConfigurator.scss

```diff
@@ -60,19 +47,3 @@
         }
```

Comment:
This does not need the 2px above to be centered, in fact the 2px makes things wonky